index.html

```diff
@@ -273,16 +287,9 @@ <h3>Concept and Initial Report</h3>
 
     <select multiple class="city-search"></select>
 
-    <div class="no-requirements-toggle-container">
-      <label class="toggle">
-        <span class="toggle-label">No Parking Requirements</span>
-        <input
-          type="checkbox"
-          class="toggle-checkbox"
-          id="no-requirements-toggle"
-        />
-        <div class="toggle-switch"></div>
-      </label>
+    <div class="city-counter-container">
+      <span id="counter-numerator">Showing 0 out of </span
+      ><span id="counter-denominator">0 cities.</span>
```

Comment:
Even better would be to only have the IDs refer to the specific numbers. That way you don't have to duplicate the other text like "Showing" and "out of" - the only thing we dynamically change is the numbers.